coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +58,23 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        original_line = str(original_file_dict[filename][
+                        source_range.start.line-1].rstrip())
+
+        # Add the new ignore using 'and' with the previous ignore.
+        if re.compile(r'(Ignore\s)(.+Bear)').search(original_line):
+            ignore_comment = re.compile(r'(.+Ignore)\s(.+Bear).*').sub(
+                r'\2 and', ignore_comment).rstrip()
+            changed_line = re.compile(r'(Ignore\s)(.+Bear)').sub(
```

Comment:
It seems you're reusing nearly the same regex everywhere. Can you try to find a "master-regex" that you can unse everywhere?